src/actions/ScanActions.tsx

```diff
@@ -252,23 +223,18 @@ export function parseScannedUri(data: string, customErrorTitle?: string, customE
 
       Actions.push('send', {
         guiMakeSpendInfo,
-        selectedWalletId,
+        selectedWalletId: coreWallet.id,
         selectedCurrencyCode: currencyCode
       })
-      // dispatch(sendConfirmationUpdateTx(parsedUri))
     } catch (error: any) {
       // INVALID URI
       setTimeout(
         () =>
-          Alert.alert(
-            customErrorTitle || s.strings.scan_invalid_address_error_title,
-            customErrorDescription || s.strings.scan_invalid_address_error_description,
-            [
-              {
-                text: s.strings.string_ok
-              }
-            ]
-          ),
+          Alert.alert(s.strings.scan_invalid_address_error_title, s.strings.scan_invalid_address_error_description, [
```

Comment:
While we are here, should we upgrade this to a buttons modal? That would match our theming better, and is almost as easy to use.

Reply:
Totally fair. I was tempted to do that and you've pushed me over the edge!